.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 yarn-error.log
+.idea
```

Comment:
If this is just something you do on your system, might be more appropriate to do something like I do:

In `/home/pepsi/.gitconfig`:

```
[core]
        excludesfile = /home/pepsi/.gitignore_global
```

In `/home/pepsi/.gitignore_global`:
```
*~
\#*
.#*
NOTES.*
```

Reply:
The main project's .gitignore has a bunch of IDE related lines:
https://github.com/jerryscript-project/jerryscript/blob/master/.gitignore#L9

I don't feel strongly about it though, although I've got bitten by system-level gitignore in the past where a repo actually did have files that were being ignored by my own system-level gitignore. So I've lived with an empty one ever since...

Reply:
My thoughts are with @martijnthe :) I feel alike

Reply:
Fine with me; I'd never seen .idea before. :)